src/OutputWriters/checkpointer.jl

```diff
@@ -29,17 +24,17 @@ end
                                :buoyancy, :closure, :velocities, :tracers, :timestepper]
                  )
 
-Construct a `Checkpointer` that checkpoints the model to a JLD2 file every so often as
-specified by `schedule`. The `model.clock.iteration` is included
-in the filename to distinguish between multiple checkpoint files.
+Construct a `Checkpointer` that checkpoints the model to a JLD2 file on `schedule.`
+The `model.clock.iteration` is included in the filename to distinguish between multiple checkpoint files.
+
+To restart or "pickup" a model from a checkpoint, specify `pickup=true` when calling `run!`, ensuring
+that the checkpoint file is the current working directory. See [`run!`](@ref) for more details.
```

Comment:
The problem with the docs not building is this ref of `run!`. Within `OutputWriters` module I don't think there is access to `run!`... When I changed this ref to
```Julia
[`Oceananigans.Simulations.run!`](@ref)
```
I got this error in the docs build:
```Julia
[ Info: CrossReferences: building cross-references.
┌ Warning: unable to get the binding for '[`Oceananigans.Simulations.run!`](@ref)' in src/appendix/library.md from expression ':(Oceananigans.Simulations.run!)' in module Oceananigans.OutputWriters
│   exception = UndefVarError: Oceananigans not defined
└ @ Documenter.CrossReferences ~/.julia/packages/Documenter/4JDQo/src/CrossReferences.jl:137
```

Reply:
That's because `run!` is not available within `OutputWriters` module. One could import it at the top, e.g., `using Oceananigans.Simulations: run!` but that can't happen because in 
https://github.com/CliMA/Oceananigans.jl/blob/bfee249e49bd4dd12e91fbc905a8860fa9d9c6e6/src/Oceananigans.jl#L191-L193
`Simulations` module is included after the `OutputWriters`. And also one can't simply reverse the order because the former depends on the latter...

Any ideas?

Reply:
I get it.

We need to document `pickup=true` in `Simulations` (not `OutputWriters`).

Reply:
There isn't even a section devoted to using `Simulation` and `run!` (or `TimeStepWizard`) yet? The docs were designed before `Simulation` existed...

Reply:
If you ref it there it'll work I think..

Reply:
Wait, I think I didn't understand exactly what you suggesting...

Reply:
My point: `run!` should be documented along with `Simulation`, but there's no documentation at all for `Simulation`!

I am very happy with your solution.